lib/BackgroundJobs/InitialContentImportJob.php

```diff
@@ -57,15 +57,15 @@ protected function run($argument): void {
 			return;
 		}
 
-		$registeredProviders = $this->configService->getProviders();
-		$identifier = ProviderConfigService::getConfigKey($providerObj->getAppId(), $providerObj->getId());
+		$registeredProviders = $this->providerService->getProviders();
+		$identifier = $this->providerService->getConfigKey($providerObj->getAppId(), $providerObj->getId());
```

Comment:
This method is static

Reply:
yeah, but the object was already there so I used it, is there a performance hit?

Reply:
Not sure. I would expect runtime warnings or somesuch

Reply:
cool, I'll change that and every other "instance"

Reply:
it's statically called in other places as well so leaving it as a static method

Reply:
Static methods have some downsides making dependency injection harder, so I'd really recommend using instance methods unless you have a reason to use a static method.

Reply:
But we don't do any dependency injection when calling static methods, it is like calling a function or am I wrong? The upside of this static method is that we won't need to inject the provider service dep whenever we need to access just this method, which is some places.